geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -526,6 +526,55 @@ def _get_geolocation_metadata(metadata):
 
 
 def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=False):
+    if metadata_only:
+        return call_single_time([fnames[0]], metadata_only=metadata_only)
```

Comment:
Should this come after the "all_metadata" bit?  So we actually return all of the metadata appropriately?

Reply:
Refactored this to return all metadata.